dataprofiler/profilers/profile_builder.py

```diff
@@ -970,15 +974,15 @@ def report(self, report_options=None):
                 "file_type": self.file_type,
                 "encoding": self.encoding,
                 "memory_size": self.memory_size,
+                "times": self.times,
             }),
             ("data_stats", OrderedDict()),
         ])
-
         report["data_stats"] = self._profile.profile
         return _prepare_report(report, output_format, omit_keys)
 
-    @staticmethod
-    def _clean_data_and_get_base_stats(data, sample_size,
+    @utils.method_timeit(name="clean_and_base_stats")
```

Comment:
I'm approving, but I dont think the average user will understand clean_and_base_stats

Reply:
i'm open to other names, i wasn't really sure what to put.

Reply:
may be `get_clean_and_base_stats `

Reply:
I'm not sure that makes it more clear to the user as the intent is to label the cost of what is being calculated but not just the function